gaming/README.md

```diff
@@ -0,0 +1,105 @@
+# Spanner Gaming Samples
+
+This repository contains sample code for the following use-cases when using Cloud Spanner for the backend:
+
+- Player creation, login, and skin changes
+
+## How to use this
+
+### Setup infrastructure
+A terraform file is provided that creates the appropriate resources for these samples.
+
+Resources that are created:
+- Spanner instance and database based on user variables in main.tfvars
+- (FUTURE) GKE cluster to run the load generators
+
+To set up the infrastructure, do the following:
+
+- Copy `infrastructure/terraform.tfvars.sample` to `infrastructure/terraform.tfvars`
```

Comment:
Side note: I love it when instructions in the README are this clear and concrete 👍

Reply:
Thanks!